std/meta.d

```diff
@@ -1177,9 +1228,13 @@ unittest
 /**
  * Creates an `AliasSeq` which repeats a type or an `AliasSeq` exactly `n` times.
  */
-template Repeat(size_t n, TList...) if (n > 0)
+template Repeat(size_t n, TList...)
 {
-    static if (n == 1)
+    static if (n == 0)
+    {
+        alias Repeat = AliasSeq!();
+    }
+    else static if (n == 1)
     {
         alias Repeat = AliasSeq!TList;
     }
```

Comment:
`else static if (n == 2){...}` is no longer necessary.
And, could you please add a test for `Repeat!(0, X)`?